client/packages/cli/src/schemas/registerAsset.ts

```diff
@@ -4,27 +4,30 @@ export const AssetRegistrationSchema = z.object({
   endpoint: z.string({
     invalid_type_error: 'Enpoint must be a string',
     required_error: 'Endpoint is required',
-  }),
+  }).startsWith('ws://'),
   dest: z.string({
-    invalid_type_error: 'Destination type must be a string',
+    invalid_type_error: 'Destination type must be a string one of the options: t0rn; local; para',
     required_error: 'Destination type is required',
-  }),
+  }).regex(/^(t0rn|local|para)$/),
```

Comment:
t.rn

Reply:
currently, it only supports t0rn since production chains would require registering assets through governance.. updated  the value to be an enum in commit [40a646a](https://github.com/t3rn/t3rn/pull/1444/commits/40a646a0d3d0d939c061237cf09ec900b8dd6553)